app/controllers/api/account.php

```diff
@@ -2446,72 +2434,79 @@
     ->label('sdk.response.model', Response::MODEL_SESSION)
     ->label('abuse-limit', 10)
     ->param('sessionId', '', new UID(), 'Session ID. Use the string \'current\' to update the current device session.')
-    ->inject('request')
+    ->param('identity', false, new Boolean(), 'Refresh OAuth2 identity. If enabled and session was created using an OAuth provider, the access token of the identity will be refreshed', true)
```

Comment:
Hmm not sure this is a good name for the parameter. Please suggest some alternatives

Reply:
This endpoint takes a `sessionID` as input. The session already contains information if it was from an OAuth provider or not right? 

Why do we need an additional parameter `identity` 

If you make a PATCH request using your OAuth session, it will update ( renew ) that specific session,

Reply:
Another opened question, should it throw exception if you want to refresh identity and session doesn't have identity?

Maybe there should be no param and we always extend both session duration and refresh oauth token?

Reply:
Param removed.